src/Uno.UI.RuntimeTests/Helpers/SizeAssertion.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using Windows.Foundation;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+
+namespace Uno.UI
+{
+	internal static class SizeAssertionExtensions
+	{
+		internal static SizeAssertion Should(this Size size, double epsilon = 0.01d) => new SizeAssertion(size, epsilon);
+
+		internal static PointAssertion Should(this Point point, double epsilon = 0.01d) => new PointAssertion(point, epsilon);
+
+		internal static RectAssertion Should(this Rect rect, double epsilon = 0.01d) => new RectAssertion(rect, epsilon);
+
+		internal static (bool isDifferent, double difference) CheckDifference(double value1, double value2, double epsilon)
+		{
+			if (value1.Equals(value2))
+			{
+				return (false, 0d);
+			}
+
+			var difference = Math.Abs(value1 - value2);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'difference'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5037645)